src/mlb.py

```diff
@@ -0,0 +1,27 @@
+from sklearn.preprocessing import MultiLabelBinarizer
+from sklearn.multiclass import OneVsRestClassifier
+from sklearn.linear_model import LogisticRegression, RidgeClassifier
+
+def get_mlb(tags_counts, y_train, y_val):
+    mlb = MultiLabelBinarizer(classes=sorted(tags_counts.keys()))
+    y_train = mlb.fit_transform(y_train)
+    y_val = mlb.fit_transform(y_val)
+
+    return mlb, y_train, y_val
+
+
+def train_classifier(X_train, y_train, penalty='l1', C=1):
+    """
```

Comment:
Maybe make all docstrings (for the functions which they are useful to have) a bit nicer